explainaboard/analysis/analyses.py

```diff
@@ -391,11 +396,18 @@ def perform(
             if x not in cases[0].features:
                 raise RuntimeError(f"combo analysis: feature {x} not found.")
 
-        combo_map: dict[tuple[str, ...], int] = {}
+        combo_map: dict[tuple[str, ...], list] = {}
         for case in cases:
             feat_vals = tuple([case.features[x] for x in self.features])
-            combo_map[feat_vals] = combo_map.get(feat_vals, 0) + 1
-        combo_list = list(combo_map.items())
+            if feat_vals not in combo_map:
+                combo_map[feat_vals] = [0, []]
+            combo_map[feat_vals][0] += 1
+            combo_map[feat_vals][1].append(case.sample_id)
+
+        combo_list = [
+            (k, v[0], self._subsample_analysis_cases(self.sample_limit, v[1]))
+            for k, v in combo_map.items()
+        ]
```

Comment:
With the suggestions above:
```suggestion
            combo_map[feat_vals].append(case.sample_id)
        
        combo_list = [
            ComboOccurence(k, self._subsample_analysis_cases(self.sample_limit, v)
            for k, v in combo_map.items()
        ]
```